<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QThreadPoolDemo</class>
 <widget class="QWidget" name="QThreadPoolDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_14">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_15">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_16">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
